src/pages/components/Components.tsx

```diff
@@ -63,8 +70,8 @@ const Components = (): JSX.Element => {
         ]
       },
       {
-        text: 'Dropdown menu',
-        to: basePath + '/dropdown-menu'
+        text: 'Dropdown',
```

Comment:
thoughts on making these name changes in SB instead of docs? we can also check with catie to see what they should be and make sure they're aligned in design libraries too